themes/bootstrap3/templates/Recommend/SideFacets.phtml

```diff
@@ -20,7 +32,7 @@
 <?php $checkboxesShown = false; ?>
 <?php if (count($checkboxFilters) > 0):
     foreach ($checkboxFilters as $current) {
-      if ($results->getResultTotal() > 0 || $current['selected'] || $current['alwaysVisible']) {
+      if ($results->getResultTotal() > 0 || $current['alwaysVisible']) {
```

Comment:
@EreMaijala, I noticed that if I selected a combination of checkboxes that cause the result set to be empty, I ended up with a redundant display like this:

![image](https://user-images.githubusercontent.com/309069/52347332-a85b7780-29ef-11e9-9b48-d3cbdde33df1.png)

This small adjustment eliminates the duplicate side checkboxes. Any objections?

Reply:
@EreMaijala, there is one case where side facets are still necessary: some services (such as Summon) include checkbox filters that actually expand the result set (e.g. "Include results from other libraries"). We need a way to display these controls on the no results screen, and side facets is currently the only mechanism to do that.

Reply:
Right, then I think this is fine as it is.